colossalai/graph/passes/adding_split_node_pass.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+from torch.fx import symbolic_trace
+from torch.fx.node import Node
+from torch.fx.passes.split_module import split_module
+
+
+def pipe_split():
+    pass
```

Comment:
why pass?

Reply:
This function is just used to create a node on graph to annotate the split location. The split node will be erased after split_module pass, so it should never be used during runtime.